src/test/huffman_bench.cpp

```diff
@@ -1,14 +1,15 @@
 #include "src/huffman.hpp"
 #include "src/version/version.hpp"
 
+#include <array>
 #include <benchmark/benchmark.h>
 
 namespace {
 
 void BM_CodeTable(benchmark::State& state)
 {
-  const auto frequencies = std::vector<std::pair<char, std::size_t>>{
-      {'e', 100}, {'n', 20}, {'x', 1}, {'i', 40}, {'q', 3}};
+  constexpr auto frequencies = std::array<std::pair<char, std::size_t>, 5>{
```

Comment:
Does this change the benchmark results? If so, then maybe benchmark both std::array and a non-statically-size input separately?

Reply:
Possibly? I haven't looked into this in detail.

I changed this to take advantage of CTAD but that isn't too necessary. I'll restore the use of `std::vector`.